NAMESPACE

```diff
@@ -8,6 +8,7 @@ import(Biobase)
 import(gdsfmt)
 import(graph)
```

Comment:
i'm not sure that we're using stuff from the `graph` package anymore. I think that was being used in the old `pcair` function. I don't think it's being used anywhere else.

Reply:
removed graph from DESCRIPTION and NAMESPACE